neo/rawio/spikeglxrawio.py

```diff
@@ -109,6 +112,12 @@ def __init__(self, dirname="", load_sync_channel=False, load_channel_location=Fa
         BaseRawWithBufferApiIO.__init__(self)
         self.dirname = dirname
         self.load_sync_channel = load_sync_channel
+        if load_sync_channel:
+            warn(
+                "The load_sync_channel=True option is deprecated and will be removed in version 0.15 \n"
+                "The sync channel is now loaded as a separate stream by default and should be accessed as such. ",
```

Comment:
This is still a little unclear if you're going to still give me the sync channel as the last channel or not during the deprecation period. 
```suggestion
                "The sync channel is now loaded as a separate stream by default and should be accessed as such. ",
```